litestar/stores/file.py

```diff
@@ -28,15 +28,22 @@ def _safe_file_name(name: str) -> str:
 class FileStore(NamespacedStore):
     """File based, thread and process safe, asynchronous key/value store."""
 
-    __slots__ = {"path": "file path"}
+    __slots__ = {"path": "file path", "_create_folders": "flag to create folders of path"}
```

Comment:
is this meant to be `create_folders` instead?

Reply:
Actually I see the dictionary in slots for the first time ever, so I don't really know. I used private attribute notation, and expected the slot entry to match the attribute name `self._create_folders`

Reply:
Doesn't this duplicate the docstr overall?